lib/fluent/plugin/out_copy_ex.rb

```diff
@@ -58,7 +58,11 @@ def emit(tag, es, chain)
       @outputs.each_index do |idx|
         _es = @deep_copy ? es.dup : es
         begin
-          @outputs[idx].emit(tag, _es, NullOutputChain.instance)
+          if @outputs[idx].respond_to?(:emit_events)
```

Comment:
I've tried to create proc object in this commit: https://github.com/sonots/fluent-plugin-copy_ex/pull/3/commits/5fe79eeb988d12975fd91cd96502f9f9b417b765
Could you take a look?